src/app/components/Image/BlurredBackground.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './index.styles';
+
+type BlurredBackgroundProps = {
+  src?: string;
+};
+
+const BlurredBackground = ({ src }: BlurredBackgroundProps) => {
```

Comment:
I think the blurred background span always loads immediately (no lazy/defer), so offscreen portrait promos might still trigger extra `iChef` fetches. Should we tie it to the same lazy/intersection logic as the main image?

Reply:
You're right about the span always loading, need to think about the implementation a bit. I'd rather not make an intersection observer for every promo on a given page, so figuring out where to put this (probably grid level than page level?) is also something that might be annoying.

In a way, this might still be a fine trade-off since the blurred background essentially acts as a new, slightly better looking but less performant placeholder for promos with portrait images.

Reply:
Good point!

To avoid another intersection observer per promo could we piggyback on the same observer we use for view tracking (which observes each promo)? and only set the blurred background when it becomes visible. 

No need to do that hook up in this PR, you could leave it eagerly loaded for now and add a follow-up to hydrate it via the shared observer - up to you